src/format/parser.d

```diff
@@ -2503,6 +2511,19 @@ private:
 		assert(0);
 	}
 
+	void parseShortenedFunctionBody() in(match(TokenType.FatArrow)) {
+		auto spanGuard = spliceSpan();
```

Comment:
Does it change something if you remove this? There might be a span here already, but if there isn't this is good to have.